ext/gd/gd.c

```diff
@@ -821,6 +821,28 @@ ZEND_BEGIN_ARG_INFO(arginfo_imagesetinterpolation, 0)
 	ZEND_ARG_INFO(0, method)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_INFO(arginfo_imagegifanimbegin, 0)
+	ZEND_ARG_INFO(0, im)
```

Comment:
Doesnt the 0 means optional arg but the comment at the function body declares this as reqired arg?